src/lib/Layout/PanelSideBarLayout/PanelSideBarLayout.tsx

```diff
@@ -75,8 +76,8 @@ export const PanelSideBarLayout = (props: PanelSideBarLayoutProps) => {
         </Nav>
       </nav>
 
-      <section className={classNames({ toggled: !isOpen })}>
-        <PanelSideBar toggledSidebar={isOpen} />
+      <section className={`${renderFirstItemsLevelAsTiles ? "section-tiles" : "section-no-tiles"} ${classNames({ toggled: !isOpen })}`}>
```

Comment:
couldn't you add the new one also over the classNames function? or doesnt makes sense? 🤔 (to force also future changes more to use the classNames 😄)
https://www.npmjs.com/package/classnames

Reply:
@neotob Done with classNames instead, I thought it was better without for the if else, but at the end it's the same.. hope it's fine for you :D